tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -21,6 +21,11 @@ test.describe('Choose Template Type Page', () => {
     await expect(chooseTemplatePage.pageHeader).toHaveText(
       'Choose a template type to create'
     );
+
+    await expect(chooseTemplatePage.learnMoreLink).toHaveAttribute(
```

Comment:
As the learnMoreLink is using the testid locator rather than text itself, it might be a good idea to add in an expect for the text of the link like the pageHeader has. 🤔

Reply:
Would you prefer it to find the link by text and then check the href value?

Reply:
it would work either way tbh.  I think Playwright usually recommend using text as locator as it does check the text at the same time but either way would be fine.

Reply:
I've changed to find by text and removed data-testid from the link. I think it's cleaner.